install.sh

```diff
@@ -132,9 +132,9 @@ main() {
 
   tar xzf "$DLPATH" -C "$DIR"
 
-  $SUDO cp -f "$DIR"/gpud /usr/sbin
```

Comment:
Our default systemd relies on `/usr/sbin/gpud`. Any reason why we need `/usr/local/bin`?

Reply:
Oh

> Changes the binary install path from /usr/sbin to /usr/local/bin to support immutable systems like Fedora CoreOS and OpenShift, where /usr/sbin is read-only.